parquet/src/test/java/org/apache/iceberg/parquet/TestParquet.java

```diff
@@ -127,4 +140,80 @@ public void testNumberOfBytesWritten() throws IOException {
         records.toArray(new GenericData.Record[]{}));
     return Pair.of(file, size);
   }
+
+  @Test
+  public void testBloomFilterWriteRead() throws IOException {
+    File parquetFile = generateFileWithBloomFilter();
+
+    try (ParquetFileReader reader = ParquetFileReader.open(ParquetIO.file(localInput(parquetFile)))) {
+      BlockMetaData rowGroup = reader.getRowGroups().get(0);
+      BloomFilterReader bloomFilterDataReader = reader.getBloomFilterDataReader(rowGroup);
+
+      ColumnChunkMetaData intColumn = rowGroup.getColumns().get(0);
+      BloomFilter intBloomFilter = bloomFilterDataReader.readBloomFilter(intColumn);
+      Assert.assertTrue(intBloomFilter.findHash(intBloomFilter.hash(30)));
```

Comment:
I think we should add in a few negative tests as well since all of these check for inclusion and none for exclusion. I'm not super worried about this since I assume this code path is already well tested in Parquet.

Reply:
The hash values might conflict so I didn't add exclusion tests.